cmd/main.go

```diff
@@ -69,7 +72,7 @@ func main() {
 	var probeAddr string
 	var configMap string
 	var enabledServices controllers.EnabledServices
-	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":9443", "The address and port the metric endpoint binds to.")
```

Comment:
Switching the default value here to 9443 may conflict with the webhook port although we don't have a webhook server yet. Actually, the metric port is not exposed either.